cli/types/cypress.d.ts

```diff
@@ -54,16 +54,12 @@ declare namespace Cypress {
   }
 
   interface RemoteState {
-    auth?: {
-      username: string
-      password: string
-    }
+    auth?: Auth
     domainName: string
     strategy: 'file' | 'http'
     origin: string
-    fileServer: string
+    fileServer: string | null
     props: Record<string, any>
-    visiting: string
```

Comment:
It feels non-passive to remove this key. Is this only used internally or will users be receiving an object with these types?

Reply:
`config.remote` is a non-documented property of config. It's not even [in here](https://github.com/cypress-io/cypress/blob/develop/packages/config/lib/options.ts). As far as I can tell, we only use it internally, so it and this RemoteState interface should probably be moved to internal types.

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/21137/commits/7407dcbaf0878731de905736f2956eddfe5e5f6c